lib/src/audio_waveforms.dart

```diff
@@ -41,17 +41,20 @@ class _AudioWaveformsState extends State<AudioWaveforms> {
   double _initialOffsetPosition = 0.0;
   double _initialPosition = 0.0;
 
+  late VoidCallback recorderControllerListener;
```

Comment:
Just create a normal function and add it the `addListener` and `removeListener`.